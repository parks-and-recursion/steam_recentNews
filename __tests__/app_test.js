import React from 'react'
import App from '../client/src/app.jsx'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

describe('App', () => {
	it('matches the snapshot', () => {
		const tree = renderer.create(<App />).toJSON()

		expect(tree).toMatchSnapshot()
	})

	// it('should have a header RECENT UPDATES', () => {
	// 	const wrapper = shallow(<View />)
	// 	const text = wrapper.find('Header').text()
	// 	expect(text).toEqual('RECENT UPDATES')
	// })
})