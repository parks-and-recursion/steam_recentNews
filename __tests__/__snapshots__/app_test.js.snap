// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches the snapshot 1`] = `
<h1>
  See it here
</h1>
`;
